### PR TITLE
Small priority queue fixes

### PR DESCRIPTION
**Pull request by Ihromant**

### CODE REVIEW

File: classlib/src/main/java/org/teavm/classlib/java/util/TPriorityQueue.java [hunk 1]
```diff
@@ -112,6 +112,7 @@ public boolean offer(E e) {
         if (e == null) {
             throw new TNullPointerException();
         }
+        comparator.compare(e, e);
```
Comment (konsoletyper) [inline classlib/src/main/java/org/teavm/classlib/java/util/TPriorityQueue.java:115]: What's the purpose of this call?

Comment (Ihromant) [reply classlib/src/main/java/org/teavm/classlib/java/util/TPriorityQueue.java:115]: If we add not consistent element to queue of size 0 or 1 - it successfully adds and then we have broken state.

Comment (konsoletyper) [reply classlib/src/main/java/org/teavm/classlib/java/util/TPriorityQueue.java:115]: What do you mean by "not consistent element"?

Comment (Ihromant) [reply classlib/src/main/java/org/teavm/classlib/java/util/TPriorityQueue.java]: Ok, you're right, this check not needed.

**Pull request merged**